bcda/api/requests_test.go

```diff
@@ -303,3 +482,22 @@ func (s *RequestsTestSuite) genASRequest() *http.Request {
 
 	return req.WithContext(ctx)
 }
+
+func (s *RequestsTestSuite) genGetJobsRequest(statuses []models.JobStatus) *http.Request {
+	target := "http://bcda.cms.gov/api/v2/jobs"
```

Comment:
should this always be using v2?

Reply:
It doesnt *really* matter but I think its good to be consistent when it comes to the test. Will update.